File: frontend/src/lib/bankAccount.ts
/**
 * Static bank-transfer details shown on the publish-fee Pay step while the
 * payment-gateway (Moyasar) integration is not yet live. Sellers transfer the
 * fee to this account and upload a receipt for admin review.
 *
 * The QR image (public/bank-account.jpeg) already encodes these same details
 * for in-app banking scan-to-pay.
 */
export const BANK_ACCOUNT = {
  bankName: 'مصرف الراجحي',
  accountName: 'مؤسسة برق واضح',
  accountNumber: '433000010006086240365',
  iban: 'SA9080000433608016240365',
  qrImage: '/bank-account.jpeg',
} as const;

File: mobile/lib/core/constants/bank_account.dart
/// Static bank-transfer details for paying the after-sale commission while the
/// payment gateways are not yet live. The QR asset encodes the same details for
/// scan-to-pay from a banking app.
class BankAccount {
  static const String bankName = 'مصرف الراجحي';
  static const String accountName = 'مؤسسة برق واضح';
  static const String accountNumber = '433000010006086240365';
  static const String iban = 'SA9080000433608016240365';
  static const String qrAsset = 'assets/images/bank_account.jpeg';
}
